stores/gaugeStore.js

```diff
@@ -123,10 +123,10 @@ class Store {
           logo: '/moonriver.png',
           url: '',
           chainId: 1285,
-          gaugeProxyAddress: '0xAc8204a9d79CA87D192ea98A9381600642A66a5F',
-          mirroredVotingEscrow: '0x1747D329CB37e0A0f387f24065aDdbc60eAB69DD',
+          gaugeProxyAddress: '0xca78ca5C3Da9a5a4C960C1757456E99d9F1bc76d',
```

Comment:
config should not change

Reply:
i think these were merged changes from staging